tools/Vrops.py

```diff
@@ -142,7 +142,12 @@ def get_hosts(self, target, token, parent_uuids):
         return self.get_resources(target, token, parent_uuids, resourcekinds=["HostSystem"])
 
     def get_vms(self, target, token, parent_uuids):
-        return self.get_resources(target, token, parent_uuids, resourcekinds=["VirtualMachine"], data_receiving=True)
+        uuids_chunked = list(chunk_list(parent_uuids, 150))
```

Comment:
Why 150? Shouldn't they be chunked at 1000?

Reply:
We decreasing the amount of parent uuids here. Hosts to be precise.